### PR TITLE
Speculative fix for bug in weblab footer

### PR DESCRIPTION
**Pull request by bethanyaconnor**
A user is seeing the channel id being added twice to the link in the weblab footer. I can't reproduce the issue, even on a project they're seeing the issue on, but this seems like a possible fix to the issue.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by hannahbergam — COMMENTED** [review]
It's so weird that we can't reproduce it... it makes me feel on the fence about merging a fix. I am also hoping that my abuse link fix for weblab would still work in this case because I left the usual channel id pattern in the parsing function, but it's hard to tell without being able to actually test it. Has the user reported the same issue with abuse reporting?

**Comment by bethanyaconnor** [comment]
The problem is that it 404s so I'm not sure if the abuse link would work either?

File: dashboard/public/weblab/footer.js [hunk 1]
```diff
@@ -30,10 +30,14 @@ var applyFooter = function() {
   }
   if (window.encrypted_channel_id) {
     var viewCodeLink = document.getElementById("pagefooter_view_code");
-    viewCodeLink.href =
-      viewCodeLink.href + "/" + window.encrypted_channel_id + "/view";
+    if (!viewCodeLink.includes(window.encryped_channel_id)) {
```
Comment (bethanyaconnor) [inline dashboard/public/weblab/footer.js]: Also for future me: this is misspelled 🤦‍♀️

Comment (hannahbergam) [reply dashboard/public/weblab/footer.js]: oh good catch!

**Pull request closed without merging**